src/main/java/sungshin/sooon/controller/BoardController.java

```diff
@@ -0,0 +1,66 @@
+package sungshin.sooon.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.data.domain.Pageable;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import sungshin.sooon.dto.PostRequestDto;
+import sungshin.sooon.dto.PostResponseDto;
+import sungshin.sooon.dto.PostUpdateRequestDto;
+import sungshin.sooon.model.Post;
+import sungshin.sooon.service.BoardService;
+
+import java.security.Principal;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/board")
+public class BoardController {
+
+    private final BoardService boardService;
+
+    // 전체 글 리스트
+    @GetMapping("/list")
+    public Page<Post> list(@PageableDefault(page = 1, sort = "created_at", direction = Sort.Direction.DESC) Pageable pageable, Model model) {
+        Page<Post> postList = boardService.getPostList(pageable);
+        model.addAttribute("postList", postList);
+
+        return boardService.getPostList(pageable);
+    }
+
+    // 게시글 저장
+    @PostMapping("/post")
+    public Long createPost(@RequestBody PostRequestDto postRequestDto) {
+        return boardService.savePost(postRequestDto);
+    }
+
+    // 게시글 상세
+    @GetMapping("/detail/{post_id}")
+    public PostResponseDto getBoardInformation(@PathVariable("post_id") Long post_id) {
```

Comment:
나중에 개발 컨벤션 정하면 되긴하는데 자바는 카멜케이스가 컨벤션으로 알고 있어서 통일하면 좋을 것 같아요
```suggestion
    public PostResponseDto getBoardInformation(@PathVariable("post_id") Long postId) {
```